ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala

```diff
@@ -96,7 +94,7 @@ private[testtool] final class ParticipantTestContext private[participant] (
     val identifierSuffix: String,
     referenceOffset: LedgerOffset,
     services: LedgerServices,
-    commandTtlFactor: Double)(implicit ec: ExecutionContext) {
+    ttlNanos: Long)(implicit ec: ExecutionContext) {
```

Comment:
Preferably we should use `Duration` here.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3052/commits/f28736e5df07b0d30fd1b1d174e63fa974c75fde